### ISSUE TITLE
Why there's no use of val.list

### ISSUE THREAD

**Issue by eclinyuru** [charlesyou999648/MONA#4]
Hello! I noticed that  it seems that the val.list  isn't being used. I am wondering if you could explain the reason，since I've always thought that the training process needs to use val.list🥺 thanks a lot！

**Comment by charlesyou999648** [comment]
Hi, thank you for your question!

In this particular implementation, the validation set isn’t read from `val.list`. Instead, the validation process is managed internally -- it is only used for monitoring performance, tuning hyperparameters, and selecting checkpoints, not for updating model weights -- so the file isn’t explicitly used. That’s why you don’t see it being called in the code.
